packages/components/fields/multiline-text-field/src/multiline-text-field.tsx

```diff
@@ -0,0 +1,222 @@
+import {
+  ChangeEventHandler,
+  Component,
+  FocusEventHandler,
+  ReactElement,
+  ReactNode,
+} from 'react';
+import {
+  createSequentialId,
+  filterDataAttributes,
+  getFieldId,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Spacings from '@commercetools-uikit/spacings';
+import FieldLabel from '@commercetools-uikit/field-label';
+import MultilineTextInput from '@commercetools-uikit/multiline-text-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+
+const sequentialId = createSequentialId('multiline-text-field-');
+
+const hasErrors = (errors: { [s: string]: unknown } | ArrayLike<unknown>) =>
+  errors && Object.values(errors).some(Boolean);
+
+type TErrorRenderer = (key: string, error?: boolean) => ReactNode;
+
+export type TMultiTextFieldProps = {
+  // FieldLabel
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id: string;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  /**
+   * A map of errors. Error messages for known errors are rendered automatically.
+   * <br />
+   * Unknown errors will be forwarded to `renderError`
+   */
+  errors: {
+    missing: boolean;
+  };
+  /**
+   * Called with custom errors. This function can return a message which will be wrapped in an ErrorMessage. It can also return null to show no error.
+   * <br />
+   * Signature: `(key, error) => React.node`
+   */
+  renderError?: TErrorRenderer;
+  /**
+   * Indicates if the value is required. Shows an the "required asterisk" if so.
+   */
+  isRequired: boolean;
+  /**
+   * Indicates whether the field was touched. Errors will only be shown when the field was touched.
+   */
+  touched: boolean;
+
+  // TextInput
+  /**
+   * Used as HTML `autocomplete` property
+   */
+  autoComplete: string;
+  /**
+   * Used as HTML name of the input component. property
+   */
+  name: string;
+  /**
+   * Value of the input component.
+   */
+  value: string;
+  /**
+   * Called with an event containing the new value. Required when input is not read only. Parent should pass it back as value.
+   * <br />
+   * Signature: `(event) => void`
+   */
+  onChange?: ChangeEventHandler;
```

Comment:
RequiredIf was used in proptypes. 
For requiredIf, you need to show a warning if the condition is not met.

Reply:
import { warning } from '@commercetools-uikit/utils';